testsupport/wait/host.go

```diff
@@ -539,6 +539,19 @@ func UntilUserSignupHasStateLabel(expected string) UserSignupWaitCriterion {
 	}
 }
 
+// UntilUserSignupHasCompliantUsername returns a `UserSignupWaitCriterion` which checks that the given
+// UserSignup has a `.status.compliantUsername` value
```

Comment:
```suggestion
// UserSignup has a `.Status.CompliantUsername` value
```

Reply:
I used the same syntax as in the `jq` queries, but I'm fine with upper-casing here, since to match with the field names.